compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -503,8 +553,8 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
         continue;
 
       // Check if this is bias (bias is quantized later)
```

Comment:
Maybe need to change this comment too

Reply:
This comment does not have to be changed.